### PR TITLE
attempt to fix issue 5

### PR DESCRIPTION
**Pull request by tgy**

### CODE REVIEW

**Comment by RenaudWasTaken** [comment]
This compiles and passes all tests.

however it would be great to have some tests

**Comment by tgy** [comment]
That's right, good point. I will write some tests and add them to this pull request.

File: src/tests/test-helpers.hh [hunk 1]
```diff
@@ -40,4 +43,39 @@ protected:
...
+	players[0].api = new Api(st, players_ptr->players[0]);
+	players[1].id = PLAYER_ID_2;
+	players[1].api = new Api(st, players_ptr->players[1]);
+  }
+
+  virtual void TearDown()
+  {
+	delete players[0].api;
+	delete players[1].api;
+    delete st;
```
Comment (RenaudWasTaken) [inline src/tests/test-helpers.hh]: Coding style seems off

**Comment by tgy** [comment]
I added a few tests. I don't think it is worth taking the time to test every possible combination of MUR positions or other stock values. From what I see here, the output of the `mur_compute_stock_loss` API function is correct.

**Comment by tgy** [comment]
I fixed the coding message and the coding style.

**Comment by RenaudWasTaken** [comment]
pushing a travis file to check if PR passes and I'll validate it

**Comment by RenaudWasTaken** [comment]
Can you add the .travis.yml to your PR ?

**Comment by tgy** [comment]
All done.

**Pull request closed without merging**